.github/workflows/cicd.yml

```diff
@@ -1,5 +1,5 @@
 ---
-name: CI/CD
+name: CI
```

Comment:
File should be renamed i.m.o., it should reflect the workflows name. If this workflow isn't responsible for the Continues Delivery it should be named `ci.yml`